HAML + gettext conversion - the rest

himdel opened the pull request:
The last erb->haml conversion.
(Except for ./assets/javascripts/dynatree_1.2.4/skin-cfme/ui.dynatree.css.erb, ./assets/stylesheets/template.css.erb, and PR#1710)

Please note that none of the _ldap_* files in this batch were actually tested, because they are not reachable at the moment.

martinpovolny commented on vmdb/app/views/ops/_ldap_domain_show.html.haml:
```diff
@@ -0,0 +1,101 @@
+      %tr
+        %td.key= _("User Type")
+        %td.wide
+          - case @selected_ld.user_type
```
can't we here avoid the assignment to `user_type` and just leave the case as an expression?

martinpovolny commented:
I have restarted the travis, now green. Also the code looks good. 

@romanblanco, @jzigmund : you did work witn @himdel on finding these in the UI. Can you verify the changes in the UI, and drop me here a note, please?

himdel commented:
Rebased, fixed the user_type asssignment, and snuck in a random fix to @skateman's _report_list - assuming `no = _("Yes")` was not really intentional.

Some changes from @romanblanco will be coming shortly.

romanblanco commented:
@martinpovolny Checked my part. Found some issues and discussed with @himdel. Should be OK now.

himdel commented:
Added fixes from rblanco, some missing gettext and useless assignment removal elsewhere...

We still have a few files with something like `if condition ||<newline>` which haml-lint carps about (unexpected `kEND`), I'll check these out and add here too.

himdel commented:
OK.. about `if condition ||<newline>`...

    - if foo ||
    - bar
      output

is fine, however, haml-lint can't handle it.

OTOH

  - if foo ||
    bar
    output

Is wrong (outputs bar).

I fixed the 1 file which was wrong, but not to haml-lint's satisfaction. Now we can either fix haml-lint/RuboCop or merge the lines in these conditions. @skateman ? Anyone?

Only 3 files are affected:
    
    layouts/_dhtmlxlayout_explorer.html.haml
    95:    - if (@layout == "chargeback" && (x_node == "root"                                               ||
    96:    -     x_active_tree == :cb_assignments_tree || (x_active_tree == :cb_reports_tree && !@report))) ||
    97-    -    ["miq_capacity_bottlenecks"].include?(@layout)
    98-      #{layoutb_name}.cells("a").collapse();
    
    layouts/_adv_search_footer.html.haml
    37:      - if (session[:userrole] == "super_administrator" || session[:userrole] == "administrator") ||
    38-      -    (session[:userrole] != "super_administrator" && session[:userrole] != "administrator" &&
    39-      -    @edit[@expkey][:selected][:typ] == "user")
    
    miq_ae_class/_class_props.html.haml
    2:  - if x_node.split('-')[0] == "aec" || params[:pressed] ||
    3:  -    ["field_select","field_accept","field_delete"].include?(params[:action]) ||
    4-  -    (x_node.split('-')[0] == "aen" && params[:button] == "reset")
    5-    - if !@in_a_form_props

(in vm_common/_console_vmrc.html.haml it's in :javascript block and therefore fine)
